b2b-sample/guardio-insurance-sample-application/components/settingsComponents/identity-providers/identity-providers.js

```diff
@@ -291,9 +291,9 @@ const FacebookIdentityProvider = ({ onFormValuesChange, formValues }) => {
     return (
         <Form onChange={onFormValuesChange} formValue={formValues}>
             <Form.Group controlId="application_name">
-                <Form.ControlLabel>Application Name</Form.ControlLabel>
+                <Form.ControlLabel>Idp Name</Form.ControlLabel>
                 <Form.Control name="application_name" />
```

Comment:
Why this form control name leave as application_name?

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/75f1d1f961472868e3f94fe91c6db88ff55c2644